docs/design/autoware-interfaces/components/perception-interface.md

```diff
@@ -0,0 +1,70 @@
+# Perception
+
+![Node diagram](images/Perception-Bus-ODD-Architecture.drawio.svg)
```

Comment:
Maybe we want to replace this with mermaid just like in sensing-interface [PR](https://github.com/autowarefoundation/autoware-documentation/pull/328#discussion_r1200777450)

Reply:
svg graph has been replaced with mermaid